src/Compilers/VisualBasic/Portable/Symbols/ArrayTypeSymbol.vb

```diff
@@ -585,8 +585,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 End Get
             End Property
 
+        End Class
+
+        Private NotInheritable Class MDArrayNoSizesOrBounds
```

Comment:
`HasDefaultSizesAndLowerBounds` override can be moved from `MDArray` to `MDArrayNoSizesOrBounds`.